### PR TITLE
Update docs for WebdriverIO

### PR DESCRIPTION
**Pull request by christian-bromann**
## Proposed changes

This patch updates docs to WebdriverIO commands.

## Types of changes

What types of changes does your code introduce to Appium?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation Update (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have read the [CONTRIBUTING](https://github.com/appium/appium/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla.js.foundation/appium/appium)
- [x] Lint and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)
- [x] Any dependent changes have been merged and published in downstream modules

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: commands-yml/commands/interactions/touch/tap.yml [hunk 1]
```diff
@@ -26,7 +26,11 @@ example_usage:
       await action.perform();
   javascript_wdio:
     |
-      driver.touchAction('tap');
+      browser.touchAction({
+        action: 'tap',
+        x: 30,
+        y:20
```
Comment (mykola-mokhnach) [inline commands-yml/commands/interactions/touch/tap.yml]: missing space?

**Review by mykola-mokhnach — APPROVED** [review]

**Review by KazuCocoa — APPROVED** [review]

File: commands-yml/commands/device/app/app-state.yml [hunk 2]
```diff
@@ -39,8 +40,8 @@ example_usage:
 client_docs:
   java: "https://appium.github.io/java-client/io/appium/java_client/InteractsWithApps.html#queryAppState-java.lang.String-"
   python: "https://github.com/appium/python-client/blob/master/appium/webdriver/extensions/applications.py"
-  javascript_wdio: "https://webdriver.io/docs/api/appium.html" # TODO wdio documentation link
-  javascript_wd: "https://github.com/admc/wd/blob/master/lib/commands.js" # TODO wd documentation link
+  javascript_wdio: "https://webdriver.io/docs/api/appium.html#queryappstate"
+  javascript_wd: "https://webdriver.io/docs/api/appium.html#queryappstate" # TODO wd documentation link
```
Comment (erwinheitzman) [inline commands-yml/commands/device/app/app-state.yml]: comment can go?

Comment (imurchie) [inline commands-yml/commands/device/app/app-state.yml]: I don't think this `wd` docs link should go to `webdriverio`.

**Review by imurchie — COMMENTED** [review]
Wow! Thanks Christian!

File: commands-yml/commands/device/app/app-state.yml (hunk 2 above)
Comment (christian-bromann) [reply commands-yml/commands/device/app/app-state.yml]: let me fix that

**Review by imurchie — APPROVED** [review]
Great!

**Pull request merged**